.circleci/config.yml

```diff
@@ -448,31 +448,6 @@ jobs:
             echo 'Testing docs css'
             curl -f -i 'https://parl.ai/docs/_static/css/parlai_theme.css'
 
-  check_extra_tests:
```

Comment:
Oh hmm are these tests duplicates of others, since they're being deleted?

Reply:
This was the code that parsed the [long] bits.